common/models/userState.ts

```diff
@@ -14,8 +14,8 @@ export type ScheduledAffirmationNotification = {
 
 export type UserState = {
     surveyState: PartialQol;
-    focusedDomains?: FocusedDomains;
-    chosenStrategies?: string[]; // array of Strategy.internalId (firestore document id) to reference strategies in the strategies collection
-    lastSeenAffirmations?: LastSeen, // keeps track of affirmations that have been seen/scheduled at which date
-    scheduledAffirmations?: ScheduledAffirmationNotification[], // only contains upcoming affirmation notifications
+    focusedDomains: FocusedDomains;
+    chosenStrategies: string[]; // array of Strategy.internalId (firestore document id) to reference strategies in the strategies collection
+    lastSeenAffirmations: LastSeen, // keeps track of affirmations that have been seen/scheduled at which date
+    scheduledAffirmations: ScheduledAffirmationNotification[], // only contains upcoming affirmation notifications
```

Comment:
No I purposefully changed this so that instead of being optional they will begin with empty objects or arrays. `                    focusedDomains: { domains: [], subdomains: [] },
                    chosenStrategies: [],
                    lastSeenAffirmations: {},
                    scheduledAffirmations: [],`